types/rlp_marshal.go

```diff
@@ -189,3 +189,21 @@ func (t *Transaction) MarshalRLPTo(dst []byte) []byte {
 
 	return MarshalRLPTo(t.MarshalRLPWith, dst)
 }
+
+func (t Transactions) MarshalRLPTo(dst []byte) []byte {
+	return MarshalRLPTo(t.MarshalRLPWith, dst)
+}
+
+func (t *Transactions) MarshalRLPWith(a *fastrlp.Arena) *fastrlp.Value {
```

Comment:
Can we add a unit test that would invoke `MarshalRLPWith` and `UnmarshalRLPWith`?